app/presenters/chat_channel_detail_presenter.rb

```diff
@@ -39,4 +40,9 @@ def membership_users(memberships)
       }
     end
   end
+
+  def inviation_link(chat_channel)
+    host = Rails.application.config.app_domain || "localhost:3000"
```

Comment:
For me `Rails.application.config.app_domain` returns localhost locally.
```
[4] pry(main)> Rails.application.config.app_domain
=> "localhost:3000"
```
and since you define this in `test.rb` I dont think you need this `||` here.